model/metabolites.tsv

```diff
@@ -1312,7 +1312,7 @@ mets	metsNoComp	metBiGGID	metKEGGID	metHMDBID	metChEBIID	metPubChemID	metLipidMa
 "MAM00810s"	"MAM00810"	""	"C05394"	""	"CHEBI:27453"	"440653"	""	""	"HC01440"	"HC01440"	"MNXM4339"	"m00810s"	"m00810s"
 "MAM00811c"	"MAM00811"	""	""	""	""	""	""	"CE4811"	""	"CE4811"	"MNXM164249"	"m00811c"	"m00811c"
 "MAM00812s"	"MAM00812"	""	"C05403"	""	""	""	""	""	"HC01446"	"HC01446"	"MNXM36578"	"m00812s"	"m00812s"
-"MAM00813p"	"MAM00813"	""	"C07297"	"HMDB02057"	"CHEBI:28542"	"441253"	""	""	""	"C07297"	"MNXM4346"	"m00813p"	"m00813p"
+"MAM00813p"	"MAM00813"	""	"C07297"	"HMDB02057"	"CHEBI:57291"	"441253"	""	""	""	"C07297"	"MNXM4346"	"m00813p"	"m00813p"
```

Comment:
To address the current conflict with `develop`, could you please change `MAM00813p` (in the first column) to `MAM00813x`

Reply:
done, please see https://github.com/SysBioChalmers/Human-GEM/pull/358/commits/041ad028d203e6015589d8e901f2bf0ea07bb0be